cli/crates/backend/src/api/consts.rs

```diff
@@ -1,7 +1,7 @@
 pub const CREDENTIALS_FILE: &str = "credentials.json";
 pub const PROJECT_METADATA_FILE: &str = "project.json";
-pub const AUTH_URL: &str = "https://grafbase.com/auth/cli";
-pub const API_URL: &str = "https://api.grafbase.com/graphql";
+pub const AUTH_URL: &str = "https://grafbase.dev/auth/cli";
+pub const API_URL: &str = "https://api.grafbase.dev/graphql";
```

Comment:
Not relevant for the review, but for my education, what is the background behind the two TLDs?

Reply:
This was a temporary change to test things locally.

Either way, here's the break down:
| Dashboard/API domain | grafbase.com | grafbase.dev         |
|----------------------|--------------|----------------------|
| Domain we deploy to  | grafbase.app | grafbase-preview.dev |